modules/nw-creating-a-route.adoc

```diff
@@ -0,0 +1,73 @@
+[id="creating-a-route"]
+= Creating a route
+
+
+
+A route allows you to host your application at a public URL. It can either be secure or insecure, depending on the network security configuration of your application. An insecure route uses the basic HTTP routing protocol and exposes a service on an insecure application port.
+You can optionally define security, such as TLS, for a route. To know how to create a secure route, see https://docs.openshift.com/container-platform/4.4/networking/routes/secured-routes.html[secure routes].
+
+The following procedure
+describes how to create a simple HTTP-based route to an example web application, https://github.com/openshift/origin/tree/master/examples/hello-openshift[hello-openshift].
+
+.Prerequisites
+
+
+* You installed the `oc` command line interface.
+* You are logged in as an administrator.
+
+.Procedure
+
+. Create a project called `hello_openshift` by running the following command:
++
+[source,terminal]
+----
+$ oc new-project hello_openshift
```

Comment:
Namespaces and host names should use hyphens rather than underscores:
```suggestion
. Create a project called `hello-openshift` by running the following command:
+
[source,terminal]
----
$ oc new-project hello-openshift
```

Reply:
@Miciah Addressed.